docs/podman-system-migrate.1.md

```diff
@@ -0,0 +1,21 @@
+% podman-system-migrate(1) podman
+
+## NAME
+podman\-system\-migrate - Migrate container to a version
+
+## SYNOPSIS
+** podman system migrate**
+
+## DESCRIPTION
+** podman system migrate** migrates containers to a new podman version.
```

Comment:
As there's not version param, I'm assuming it's "the latest podman version" and not "a new podman version"?